setup/upgrade_minor.rst

```diff
@@ -21,16 +21,26 @@ There are two steps to upgrading a minor version:
 1) Update the Symfony Library via Composer
 ------------------------------------------
 
-First, you need to update Symfony by modifying your ``composer.json`` file
-to use the new version:
+First, you need to update all Symfony packages by modifying your ``composer.json`` file
+to use the new version (the list of packages may vary depending on what you've installed):
 
 .. code-block:: json
 
     {
         "...": "...",
 
         "require": {
-            "symfony/symfony": "3.4.*",
+            "symfony/asset": "^4.1",
+            "symfony/console": "^4.1",
+            "symfony/expression-language": "^4.1",
+            "symfony/form": "^4.1",
+            "symfony/framework-bundle": "^4.1",
+            "symfony/process": "^4.1",
+            "symfony/security-bundle": "^4.1",
+            "symfony/twig-bundle": "^4.1",
+            "symfony/validator": "^4.1",
+            "symfony/web-link": "^4.1",
+            "symfony/yaml": "^4.1"
```

Comment:
Because all dependencies will now use the `^4.0` format, the user actually should *not* need to update anything in their composer.json file (as `^4.0` will allow them to upgrade to 4.1). So, let's remove this. We could instead add a quick note to about this at the bottom:

> Your composer.json file should already be configured to allow your Symfony packages to be upgraded minor versions. But, if a package was *not* upgraded that should have been, check your `composer.json`